src/main/resources/templates/index.html

```diff
@@ -0,0 +1,12 @@
+
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org/">
```

Comment:
In the future, try to only stage files for commit that are relevant to the story (this might have been useful for your local testing, for example, but it's not related to your actual functionality).